### PR TITLE
Plugin csv conflict develop mv files

### PR DESCRIPTION
**Pull request by acavit**

### CODE REVIEW

File: plugins/EpadDB.js [hunk 1]
```diff
@@ -2414,46 +2416,68 @@ async function epaddb(fastify, options, done) {
...
                     });
+                  fastify.log.info(
+                    `calling prep download for project level files/folders for { project: projectid } : ${projectid} - {project_id: projectdbid} : ${projectdbid}`
+                  );
                   // eslint-disable-next-line no-await-in-loop
-                  await fastify.prepProjectDownload(
+                  const dicomPath = await fastify.prepProjectDownload(
                     request.headers.origin,
                     { project: projectid },
-                    { format: 'stream', includeAims: 'true' },
```
Comment (emelalkim) [inline plugins/EpadDB.js:2450]: should this be parametric (from plugin settings probably). we might not need aims for federated learning plugin but we probably will for other plugins

File: plugins/EpadDB.js [hunk 2]
```diff
@@ -2414,46 +2416,68 @@ async function epaddb(fastify, options, done) {
...
+                    { format: 'stream', includeAims: 'false' },
                     request.epadAuth,
                     writeStream,
                     {
                       project_id: projectdbid,
-                    }
+                    },
+                    undefined,
```
Comment (emelalkim) [inline plugins/EpadDB.js:2461]: you should remove these two undefineds. this is the new function signature (reqOrigin, params, query, epadAuth, output, whereJSON, returnFolder)

**Review by emelalkim — CHANGES_REQUESTED** [review]
you need to fix the prepProjectDownload usage (delete two undefineds). you can leave the other comment for later but add a todo

File: plugins/EpadDB.js (hunk 1 above)
Comment (acavit) [reply plugins/EpadDB.js:2450]: user selects aims and getting aims separately and in a further phase we are receiving the project's files. Getting aims via prepProjectDownload will duplicate the aims which are already in the aims folder.

**Review by acavit — COMMENTED** [review]
Removed undefineds

**Review by emelalkim — APPROVED** [review]

**Pull request merged**